frontend/libs/erxes-ui/src/modules/filter/components/Filter.tsx

```
@@ -216,9 +216,18 @@ const FilterView = ({
 
 const FilterDialog = (props: React.ComponentPropsWithoutRef<typeof Dialog>) => {
-  const { id, setOpenDialog } = useFilterContext();
+  const { id, setOpenDialog, resetFilterState } = useFilterContext();
   const openDialog = useAtomValue(openDialogState(id));
-  return <Dialog open={openDialog} onOpenChange={setOpenDialog} {...props} />;
+  return (
+    <Dialog
+      open={openDialog}
+      onOpenChange={(open) => {
+        setOpenDialog(open);
+        !open && resetFilterState();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)